style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgrey;
+  font-family: sans-serif;
+}
+
+#section1 {
+  background-image: url(img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-color: white;
+  background-size: 100%;
+  border: 2px solid white;
+  border-radius: 0 0 0 50px;
+}
+
+.first-list {
+  display: none;
+}
+
+.nav1 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: white;
+  padding-bottom: 1rem;
+  z-index: 1;
+}
+
+.text1 {
+  width: 100%;
+  padding-top: 4rem;
+}
+
+.logo {
+  margin: 1rem 0 0 3rem;
+  text-decoration: none;
+  font-size: 25px;
+  color: #6070ff;
+}
+
+.union {
+  margin: 1rem 3rem 0 0;
+}
+
+h1 {
+  font-size: 40px;
+  margin: 5rem 0 0 4rem;
+}
```

Comment:
- [ ] OPTIONAL: For this milestone you are supposed to use `flexbox` to layout items. Using margins is a very bad practice that can lead to numerous issues and bugs in future development. Please consider using `flexbox`.